composer.json

```diff
@@ -16,15 +16,15 @@
   },
   "autoload-dev": {
     "psr-4": {
-      "Tests\\Lendable\\Clock\\Unit\\": "tests/unit/",
-      "Tests\\Lendable\\Clock\\Functional\\": "tests/functional/"
+      "Tests\\Lendable\\Clock\\Unit\\": "tests/unit/"
```

Comment:
From template, there's no functional tests as a vfs is used for testing `PersistedFixedClock` as a unit test instead.